symplyphysics/laws/thermodynamics/chemical_potential_is_particle_count_derivative_of_internal_energy.py

```diff
@@ -0,0 +1,71 @@
+from sympy import Eq, Derivative, Point2D
+from symplyphysics import (
+    units,
+    dimensionless,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.core.geometry.line import two_point_function
+
+# Description
+## The chemical potential of a system is the amount of energy the system absorbs or releases
+## due to the introduction of a particle into the system, i.e. when the particle count increases
+## by one.
+
+# Law: mu = (dU/dN)_(S, V)
```

Comment:
Now I think if it was a good idea to separate laws for different kind of energies.

Reply:
In my opinion it's better to have 4 easily manageable specific laws than 1 general law.